modules/claims_api/app/models/claims_api/form_526.rb

```diff
@@ -68,10 +69,54 @@ def to_internal
       }.to_json
     end
 
+    def current_mailing_address
+      veteran[:currentMailingAddress]
```

Comment:
how does this error out if currentMailingAddress is not a hash or doesn't exist etc?

Reply:
@bastosmichael I have pushed up a fix for this, however we dont validate the any of the other field datatypes. Should we add virtus and do that?